tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/SeqScanExec.java

```diff
@@ -198,7 +201,34 @@ public void init() throws IOException {
       // for non-projected fields.
       Schema actualInSchema = scanner.isProjectable() ? projectedFields : inSchema;
 
-      this.projector = new Projector(context, actualInSchema, outSchema, plan.getTargets());
+      Target[] realTargets;
```

Comment:
It would be better if this change is extracted to one separate method like ``initializeProjector``.